test/utils.py

```diff
@@ -127,6 +128,32 @@ def get_async_test_timeout(default=5):
         return default
 
 
+async def wait_until(predicate, success_description, timeout=10):
```

Comment:
Can we use the existing wait_until method: https://github.com/mongodb/motor/blob/6498cd5a82d50a0d81b65c9ca495dcb45838fe0c/test/py35utils.py#L20

Reply:
Ha, that function actually got moved anyway as part of #156